free-4.md

```diff
@@ -0,0 +1,80 @@
+# Ingredients to Dish
+
+## The user and a language
+This section describes who the project would serve and why a language might be a
+good way to meet their needs.
+
+This language seeks to provide a way for people to know what dish they can create based off of the ingredients they have. You can think of it as the quintessential cook book. The users will be able to choose from a variety of dishes they can actually make, rather than havin to query through several recipes they may not have the ingredients for. A language would be a good way to meet their needs as they could make the most out of their ingredients rather than settling for a dish missing ingredients.
+
+### What's the need?
+_What need is met by your idea? Who are you helping? What is that person's
+experience like now? What would their experience be like if you could help 
+them?_
+
+This idea makes it significantly easier to decide on what to make for a particular meal by eliminating most recipes. This idea helps people who don't have a large variety of ingredients to still make delicious dishes. As of now, the person either finds a recipe that they do not have all the ingredients for, spend a long time finding a recipe they actually have all the ingredients for, or completely forgo a recipe and try to throw together a dish. With this DSL, they could actually find usable recipes.
+
```

Comment:
As someone who spends a lot of time looking through recipes with difficult ingredients, this sounds like a great idea. I usually end up making substitutions if I don't have something, which works with mixed success.